packages/client/lib/sync/txpool.ts

```diff
@@ -119,6 +124,11 @@ export class TxPool {
    * Log pool statistics on the given interval
    */
   private LOG_STATISTICS_INTERVAL = 20000 // ms
+  
+  /**
+   * 
+   */
+  private minGasPriceBump = 10 // percentage
```

Comment:
nice yeah i don't think this needs to be a configurable opt, just a constant, like MIN_GAS_PRICE_BUMP_PERCENT